libraries/azure-app-configuration-importer/src/internal/parsers/defaultConfigurationSettingsConverter.ts

```diff
@@ -140,13 +167,19 @@ export class DefaultConfigurationSettingsConverter implements ConfigurationSetti
  * @internal
  * */
 class FeatureFlagConfigurationSettingsConverter implements ConfigurationSettingsConverter {
-  featureManagementKeyWord: string;
-  enabledForKeyWord: string;
-  constructor(featureManagementKeyWord: string, enabledForKeyWord: string) {
-    this.featureManagementKeyWord = featureManagementKeyWord;
-    this.enabledForKeyWord = enabledForKeyWord;
+  legacySchemaFeatureManagementKeyWord: string;
+  msFmFeatureManagementKeyWord: string;
```

Comment:
nit pick: Can we pass `msFmFeatureManagementKeyWord` either as last or either as first we group the legacy keywords together?

Reply:
Updated